web/modules/custom/joinup_group_content_management/joinup_group_content_management.info.yml

```diff
@@ -0,0 +1,22 @@
+name: 'Joinup Group Content Management'
```

Comment:
I am going to accept this but I will keep an eye on it. If possible, we can refactor it a bit in the future so that it does not create such a fuss.. But it is a pity that we have the new module with similar name or purpose with other modules just for one action.

Reply:
It's possible by factoring out the special casing of the custom pages into an alter hook or event, so that it is no longer present in this code base but moved inside the custom page module. Then the circular dependency is gone and this can move inside joinup_community_content. But that is more complex than can be done in this PR, here we are just moving code around.

Reply:
I have made a followup to handle this and I have documented this in the module README as well as in the documentation of the `ChangeGroupAction` class.